code/controlconfig/controlsconfig.h

```diff
@@ -699,12 +731,73 @@ void control_config_do_frame(float frametime);
 /*!
  * @brief close config menu
  */
-void control_config_close();
+void control_config_close(bool API_Access = false);
```

Comment:
This is called in freespace.cpp at the end of the control config game state and now by the API. This is consistent to how some other UIs in the API work because once the librocket methods take over, all logic for the game state is no longer called. In UIs where something must be run through `init` we must then make the `close` method also available.

Reply:
Ok, good point.

Also, sorry if it wasn't clear from the one-liner message, but I was referring to all of the functions moved at and below this one as well.

Reply:
Ok... since the SCPUI is elsewhere and needs to handle the various UI events, the methods here do need to be made public/visible or otherwise take the long route of being declared as friend functions (yuck).

So, this checks out. 👍